content/09.evolution.md

```diff
@@ -69,6 +69,33 @@ Research in response to prior outbreaks of HCoV-borne infections, such as SARS a
 
 If not addressed, economic and environmental stressors are likely to cause future zoonotic transfer of diseases in the future [@doi:10.1126/science.abc3189].
 
+### Whom Does It Affect?
+
+#### Genetic Contributors to Susceptibility
+
+Throughout the pandemic, many hypotheses have been raised about factors that might make individuals more susceptible.
+Many risk factors, such as underlying health conditions, are related to the body's inflammatory response, as we review elsewhere [@individual-inequality].
+Here, we focus narrowly on genetic bases of differences in susceptibility or outcomes.
+In the past, the identification of genetic risk factors for a disease often followed a candidate gene approach, where a gene of interest was evaluated to identify variants that showed an association with the outcome of interest.
+While economical in terms of sequencing, this approach is prone to spurious results.<!--add citation-->
+Today, in the age of next-generation sequencing, a common approach is instead to conduct genome-wide scans, where a large number of single-nucleotide variants (SNPs) are evaluated to identify regions of the genome associated with differences in a phenotype.
+Genome-wide association scans (GWAS) in particular are a popular approach that employs this strategy.
+During COVID-19, both of these paradigms have been applied to the problem of identifying genetic correlates of disease severity.
+
+##### Candidate-Gene Approaches
+
+Many candidate genes have been investigated throughout the pandemic.
+An early study (published in April 2020) investigated a known variant in interferon-induced transmembrane protein 3 (_IFITM3_) among hospitalized patients in Beijing [@doi:10.1093/infdis/jiaa224].
+This gene and variant were selected because of a prior candidate gene study by some of the same authors that found an association with influenza severity among Chinese patients during the 2009 influenza A H1N1/09 pandemic [@doi:10.1038/ncomms2433].
+Here, they evaluated a small number (n=80) hospitalized COVID-19 patients to determine whether homozgyosity for the previously identified risk allele was associated with mild versus severe disease [@doi:10.1093/infdis/jiaa224].
+Their analysis revealed an association between the SNP and severe COVID-19 outcomes. <!--the in-line stats are different than the table? are the in line
```

Comment:
Strangely the significant one p<0.001 appears to be in a supplementary figure... is that what you are looking for...?

Reply:
Yes!!! Thank you, you're amazing. I'll see if I can figure out what they did! Also, if my view of candidate gene studies is too harsh, I'm happy to dial it back :laughing: For behavior genetics, they have a horrible legacy.